airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -32,6 +32,11 @@ Breaking changes
 
 Previously KubernetesPodOperator considered some settings from the Airflow config's ``kubernetes`` section.  Such consideration was deprecated in 4.1.0 and is now removed.  If you previously relied on the Airflow config, and you want client generation to have non-default configuration, you will need to define your configuration in an Airflow connection and set KPO to use the connection.  See kubernetes provider documentation on defining a kubernetes Airflow connection for details.
 
+Features
+~~~~~~~~
+
+Previously, ``namespace`` and ``name`` were required arguments for KubernetesPodOperator. Now, if ``name`` is not supplied, ``task_id`` will be used.  And if ``namespace`` is not supplied, and the task is running in a cluster, then the task pod's namespace will be used.
```

Comment:
mmm was name mandatory for KPO? I think it was optional because it could have been defined in pod template

Reply:
actually, had to fix something anyway so clarified here